compiler/noirc_driver/src/lib.rs

```diff
@@ -558,6 +564,11 @@ pub fn compile_no_check(
         enable_brillig_logging: options.show_brillig,
         force_brillig_output: options.force_brillig,
         print_codegen_timings: options.benchmark_codegen,
+        expression_width: if options.unbounded_codegen {
```

Comment:
We normally represent an unbounded expression size as having width `0`.

Reply:
Yeah, the however, `expression_width` determines what we ultimately target for the final transformed ACIR. I made this flag to distinguish that I want an unbounded codegen but I still want to target some expression width.

Reply:
Ah gotcha.

Reply:
Maybe we can distinguish more clearly with `codegen-width` and `target-width` or something like that.

Reply:
I think we can leave this as is for now.